examples/provider/bid/audience-network/README.md

```diff
@@ -0,0 +1,46 @@
+# audience-network-bid-delegate
+
+BidProvider for Audience Network
+
+## Usage
+
+```html
```

Comment:
I believe that, currently, header bidding via Audience Network requires a manual approval step, hence the "not authorised for header bidding" message you are experiencing. Let me get someone with the relevant rights to approve your test account,